updated CHANGELOG

aditishree1 opened the pull request:
### Packages impacted by this PR
@azure/cosmos

### Issues associated with this PR


### Describe the problem that is addressed by this PR
This PR adds v4 features in changelog.


### What are the possible designs available to address the problem? If there are more than one possible design, why was the one in this PR chosen?


### Are there test cases added in this PR? _(If not, why?)_


### Provide a list of related PRs _(if any)_


### Command used to generate this PR:**_(Applicable only to SDK release request PRs)_

### Checklists
- [ ] Added impacted package name to the issue description
- [ ] Does this PR needs any fixes in the SDK Generator?** _(If so, create an Issue in the [Autorest/typescript](https://github.com/Azure/autorest.typescript) repository and link it here)_
- [ ] Added a changelog (if necessary)

sajeetharan commented on sdk/cosmosdb/cosmos/CHANGELOG.md:
```diff
@@ -3,21 +3,123 @@
-- Added Correlated Activity Id feature in JS SDK for better troubleshooting of queries. The correlated activity id is added in the request header for all types of queries on Items. 
+
+- All versions and delete mode in change feed [#27058](https://github.com/Azure/azure-sdk-for-js/issues/27058)
+- Request Units (RU) based capping for Item/Document fetch Operations.
```
Are we exposing this to customers?

sajeetharan commented on sdk/cosmosdb/cosmos/CHANGELOG.md:
```diff
@@ -3,21 +3,123 @@
+- Computed Properties [#2472654](https://msdata.visualstudio.com/CosmosDB/_workitems/edit/2472654)
+- Composite Indexing [#21115](https://github.com/Azure/azure-sdk-for-js/issues/21115)
+- Correlated Activity Id [#2692245](https://msdata.visualstudio.com/CosmosDB/_workitems/edit/2692245)
+- Split/Merge support for Bulk API [#18682](https://github.com/Azure/azure-sdk-for-js/issues/18682)
```
Split/Merge proof

sajeetharan commented on sdk/cosmosdb/cosmos/CHANGELOG.md:
```diff
@@ -3,21 +3,123 @@
+- Composite Indexing [#21115](https://github.com/Azure/azure-sdk-for-js/issues/21115)
+- Correlated Activity Id [#2692245](https://msdata.visualstudio.com/CosmosDB/_workitems/edit/2692245)
+- Split/Merge support for Bulk API [#18682](https://github.com/Azure/azure-sdk-for-js/issues/18682)
+
```
Can we also mention about improved samples?

aditishree1 commented:
Closing this PR as it got a little messed up due to some mistakes. 
This is the fresh PR for updating Changelog -> https://github.com/Azure/azure-sdk-for-js/pull/30106. Addressing all the comments there.
